pkg/management/component.go

```diff
@@ -81,7 +81,7 @@ type OnboardingModule interface {
 	GenerateAuthToken() (keycloakb.TrustIDAuthToken, error)
 	OnboardingAlreadyCompleted(kc.UserRepresentation) (bool, error)
 	SendOnboardingEmail(ctx context.Context, accessToken string, realmName string, userID string, username string,
-		autoLoginToken keycloakb.TrustIDAuthToken, onboardingClientID string, onboardingRedirectURI string, reminder bool) error
+		autoLoginToken keycloakb.TrustIDAuthToken, onboardingClientID string, onboardingRedirectURI string, themeRealmName string, reminder bool) error
```

Comment:
As there is a new parameter, I would expect a modification to be able to retrieve this new param from the HTTP query.
Do we forgot it or am I missing something ?

Reply:
Do we have a way to restrict the theme requested to avoid anyone can send an email with the theme of any of our customer ... ?